### ISSUE TITLE
Upgrade to Vue.js 1.0

### ISSUE THREAD

**Issue by mpociot** [laravel/spark#116]
Now that Vue.js 1.0 is released we should upgrade the VueJS components.

**Comment by xtrasmal** [comment]
Vue is used for the dashboard, but just so that you have layout to play with. If you want Vue 1.0, Angular, Riot, React or whatever, then go ahead.

**Comment by mpociot** [comment]
Vue is also used for the registration process and the whole subscription swapping / validation.
It's not only layout.

This issue was meant as a friendly reminder that we need to upgrade from Vue 0.12 to Vue 1.0.0 so we don't forget it.

**Comment by actuallymab** [comment]
+1

**Comment by caketogo** [comment]
I am not sure that spark plays very well with Vue.js 1.0. 
Getting a lot of errors which I am now investigating

**Comment by mikebronner** [comment]
@mpociot Use the vue branch. To do this, install via composer:

```
        "laravel/spark": "dev-vue",
``

Then install in npm:

```
    "laravel-spark": "file:vendor/laravel/spark/resources/assets/js",
```

**Comment by freedomfflow** [comment]
Is there an update on this topic?  i.e. Spark utilizing Vue.js 1.x?  If that is something we should be doing ourselves, please let me know and any guidance much appreciated.
I'm learning Vue and using it on a new project which I've started with Spark.  Spark is using 0.12, and I'm not sure how to update it and not sure if updating it would cause problems.
I found a chrome install, a vue debugger, that requires both Vue.js 1.x and also the non-minified version of it.  
I'd love to know how to upgrade Spark to use Vue 1.x and an option to develop with the non-minified version.
Thanks!

**Comment by nickdenardis** [comment]
@freedomfflow Have you installed the most recent version of Spark? Looks like [VueJS is at `^1.0.0`](https://github.com/laravel/spark/blob/master/resources/assets/js/package.json#L24).

**Comment by freedomfflow** [comment]
@nickdenardis No, I have a version of spark using VueJS 0.12... I guess the way to upgrade is via composer?  That will update spark and thus vue.js ??

**Comment by nickdenardis** [comment]
@freedomfflow I haven't seen any documentation around upgrading an existing project with Spark, so I would proceed with caution if you decide to go that route. 

I can confirm that the newest version of Spark does use VueJS ^1.0.0.

```
$ npm ls
├─┬ laravel-spark@1.1.3
│ ├─┬ vue@1.0.13
...
```
